### ISSUE TITLE
[ALL] documentation: documentation very outdated

### ISSUE THREAD

**Issue by Yenthe666** [odoo/odoo#75493]
The official documentation at Odoo is getting really outdated..
Two samples:
1. https://www.odoo.com/documentation/14.0/applications/sales/sales/products_prices/prices/pricing.html
The documentation talks about a single sale price per product but this option was removed in V13 (https://github.com/odoo/odoo/commit/7d8df4bec192e5a2ff3341dcf43d59e06a57f9a9#diff-27daa04195ec708c3474aa54803b9083cfe8ddea3618a580d0ea8dacf1e1e9abL24)

2. Documentation in V14 about vendor bills still shows info/screenshots from V9(!) which is deprecated:
https://www.odoo.com/documentation/14.0/applications/finance/accounting/getting_started/process_overview/supplier_bill.htm

Quite some of the screenshots and docs also contain views that are no longer looking the same now since fields where moved for example.

**Comment by Yenthe666** [comment]
@Feyensv

**Comment by Feyensv** [comment]
Indeed, but as for a lot of things, we don't have enough people to do everything :D.
Documentation team is bigger than it was, and documentation updates are more frequent, but a lot of content needs to be updated or is missing (and priorities are unstable).  You're welcome to contribute (again :D) if you're inspired.  Since documentation content and code have been separated, the review process should be faster now.

Note that since documentation stuff has been moved to www.github.com/odoo/documentation, you should create your issues (and contributions) there ;).

**Comment by Yenthe666** [comment]
Yeah I agree. There used to be a time where contributing was rewarding. It was helpful, Odoo employees knew you personally and would genuinely help/thank you. At this point Odoo seems to be a unicorn growing in every single way but not really knowing how to manage anymore. As there's 1700+ people now, plenty of cashflow and less respect than ever for contributions I just don't feel like spending my lifetime on something that's not really appreciated to be honest.
